iocore/net/UnixNetProcessor.cc

```diff
@@ -101,6 +102,10 @@ UnixNetProcessor::accept_internal(Continuation *cont, int fd, AcceptOptions cons
   if (opt.accept_threads < 0) {
     REC_ReadConfigInteger(accept_threads, "proxy.config.accept_threads");
   }
+  REC_ReadConfigInteger(listen_per_thread, "proxy.config.exec_thread.listen");
+  if (accept_threads > 0 && listen_per_thread > 0) {
```

Comment:
Does it necessary to check the definition of SO_REUSEPORT ?

Reply:
Yes. I think so as ATS may be running on a kernel that doesn't support SO_REUSEPORT